src/tokens/saga.ts

```diff
@@ -262,6 +261,8 @@ export async function getCurrencyAddress(currency: Currency) {
       return contractKit.registry.addressFor(CeloContract.StableToken)
     case Currency.Euro:
       return contractKit.registry.addressFor(CeloContract.StableTokenEUR)
+    case Currency.Real:
+      return contractKit.registry.addressFor(CeloContract.StableTokenREAL) //FIXME what is the fallback address?
```

Comment:
We need to upgrade https://www.npmjs.com/package/@celo/contractkit in order to get this constant

Reply:
And I also found it'd be `StableTokenBRL` instead of `StableTokenREAL` in the SDK